kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/BaseContentService.java

```diff
@@ -11171,7 +11171,9 @@ public void setAvailability(boolean hidden, Time releaseDate, Time retractDate)
 
 		@Override
 		public void setAvailabilityInstant(boolean hidden, Instant releaseDate, Instant retractDate) {
-			setAvailability(hidden, timeService.newTime(releaseDate.toEpochMilli()), timeService.newTime(retractDate.toEpochMilli()));	
+			setAvailability(hidden,
```

Comment:
This change allows the passing of null values to `setAvailabiliy` through `setAvailabilityInstant`, which is needed to remove the date restriction. I am wondering if this should be a separate Jira or if it's fine as part of this work.

Reply:
Please put a comment explaining this change